src/status_im/protocol/web3/delivery.cljs

```diff
@@ -242,9 +243,11 @@
           ;; check each message asynchronously
           (when (should-be-retransmitted? options data)
             (try
-              (let [message' (check-ttl message type ttl-config default-ttl)
-                    callback (delivery-callback web3 post-error-callback data message')]
-                (t/post-message! web3 message' callback))
+              (let [message'  (check-ttl message type ttl-config default-ttl)
+                    message'' (assoc message' :powTarget pow-target
+                                              :powTime pow-time)
```

Comment:
Any chance we can avoid `message''` ? :)

Reply:
yes sure

Reply:
done